src/styles/styles.js

```diff
@@ -478,6 +476,62 @@ const styles = {
         marginRight: 8,
     },
 
+    expensiTextInputWrapper: {
+        height: 52,
+    },
+    expensiTextInputContainer: {
+        flex: 1,
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusNormal,
+        borderColor: themeColors.border,
+        paddingTop: 25,
+        paddingHorizontal: 12,
+        paddingVertical: 8,
+        justifyContent: 'center',
+        height: '100%',
+        backgroundColor: '#fff',
+    },
+    expensiTextInputContainerWithoutLabel: {
+        paddingTop: 0,
+        paddingBottom: 0,
+    },
+    expensiTextInputContainerOnFocus: {
```

Comment:
These should definitely also be much more generic styles to promote style reuse. Something like this:

```javascript
borderColorFocus: {
    borderColor: themeColors.borderFocus,
},
borderColorDanger: {
    borderColor: themeColors.badgeDangerBG,
},
```

Reply:
Done